src/tls13.c

```diff
@@ -11134,7 +11134,7 @@ int Tls13UpdateKeys(WOLFSSL* ssl)
        KeyUpdate. */
     /* TODO: add WOLFSSL_ERROR_ALREADY_IN_PROGRESS type of error here */
     if (ssl->options.dtls && ssl->dtls13WaitKeyUpdateAck)
-            return WOLFSSL_SUCCESS;
+        return 0;
```

Comment:
nit: this should go in the commit that refactor `wolfSSL_send_update_key`

Reply:
Changing.